woocommerce/woocommerce-android
# IPP: Use auto size with max line restriction for the labels

Closes: #8328
Closes: #8248


### Description

The PR adds autoscaling on the labels which are cut off in certain circumstances

### Testing instructions

* Run the payment flow on the store and device with TPP enabled
* Rotate the device
* Notice that the text is fully visible
---

* Run the payment flow on the store and device with TPP enabled with NFC disabled
* Notice that on the error screen the error text is fully visible even when the device font size set to a bigger size

### Images/gif

<img src=https://user-images.githubusercontent.com/4923871/218992693-44a4dcd6-5d4c-4141-bc40-ebcd7c79bdc5.png width=400 />
<img src=https://user-images.githubusercontent.com/4923871/218992698-268cfb0d-40ef-417a-a0fd-4e869111f4e0.png width=400 />
